content/student/graduate.md

```diff
@@ -0,0 +1,37 @@
+title: Выпускнику
+slug: graduate
+category: student
+tags: [_nav]
+submenu_index: 2
+type: tag_by_slug_sidebar
+---
+Диплом с отличием
+-----------------
+
+__ВЫПИСКА из приказа ректора БГЭУ № 587-А от 18.07.13 г. «Положение о проведении текущей аттестации студентов в учреждении образования «Белорусский государственный экономический университет»__
+
+Диплом с отличием выдается выпускнику, сдавшему государственный экзамен и защитившему дипломную работу с отметками «10 (десять)» и (или) «9 (девять)» баллов, получившему в течение всего срока обучения на семестровых экзаменах, дифференцированных зачетах и при защите курсовых работ не менее 75 % отметок «10 (десяsть)» и (или) «9 (девять)» баллов, по остальным дисциплинам – отметки не ниже «7 (семи)» баллов.
+
+Повторная текущая аттестация с целью повышения положительной отметки разрешается на выпускном курсе не более чем по трем учебным дисциплинам. Условиями для проведения повторной текущей аттестации являются:
+
+- наличие по повторно аттестуемым дисциплинам отметок не ниже «7 (семи)» баллов;
+- наличие по повторно аттестуемым дисциплинам одной отметки «6 (шесть)» баллов и не менее 75 % отметок «10 (десять)» и (или) «9 (девять)» баллов по результатам текущей аттестации по другим дисциплинам.
+
+Решение по повторной текущей аттестации студента с целью повышения положительной отметки принимается ректором (первым проректором) на основании заявления студента с учетом мнения декана факультета.
+
+Советы выпускникам ФМ
+---------------------
```

Comment:
Or you'd better make this line a blank one. But you still need to change the line above.